test/Nancy.Tests/project.json

```diff
@@ -1,6 +1,8 @@
 {
+    "version": "1.4.1.0",

```

Comment:
I'm suggesting that @jchannon takes a look since he implemented `DefaultObjectSerializer`. To be honest we changed the implementation, to it's current form, when we moved to core. The old implementation dependency on the binary formatter which was removed from .net core. Maybe it would even be wise to see if the binary formatter is back and make a call if we should revert back or not?